README.md

```diff
@@ -53,143 +46,157 @@ CPG Flow supports various stages of genomic data processing, from raw data inges
 
 ### 🌐 Production
 
-The production version of this API is available at **[api.werewolves-assistant.com](https://api.werewolves-assistant.com/docs)**.
-
-This API is used by the **[Werewolves Assistant Web App](https://werewolves-assistant.com)**.
-
-The production server is updated automatically with the latest version of the API when a new release is created. (When a new tag is pushed on the `main` branch)
+The production version of this API is documented at **[populationgenomics.github.io/cpg-flow](https://populationgenomics.github.io/cpg-flow/main/cpg_flow.html)**.
 
 ### 🛠️ Development
 
-The development version of this API is available at **[preprod.api.werewolves-assistant.com](https://preprod.api.werewolves-assistant.com/docs)**.
-
-This API is used by the **[Werewolves Assistant Web App](https://preprod.werewolves-assistant.com)**.
+The development version of this API is available at **[populationgenomics.github.io/cpg-flow](https://populationgenomics.github.io/cpg-flow/alpha/cpg_flow.html)**.
 
-The development server is updated automatically when a commit is pushed on the `develop` branch.
+The documentation is updated automatically when a commit is pushed on the `alpha` (prerelease) or `main` (release) branch.
 
 ## <a name="installation">🔨 Installation</a>
 
 To install this project, you will need to have on your machine :
 
-![Node](https://img.shields.io/badge/-nodejs-black?style=for-the-badge&logoColor=white&logo=node.js&color=366A31)
-![PNPM](https://img.shields.io/badge/-pnpm-black?style=for-the-badge&logoColor=white&logo=pnpm&color=B76507)
-![Docker](https://img.shields.io/badge/-Docker-black?style=for-the-badge&logoColor=white&logo=docker&color=004EA2)
+![PyPI](https://img.shields.io/badge/-PyPI-black?style=for-the-badge&logoColor=white&logo=pypi&color=3776AB)
+![pip](https://img.shields.io/badge/-pip-black?style=for-the-badge&logoColor=white&logo=pip&color=3776AB)
+![uv](https://img.shields.io/badge/-uv-black?style=for-the-badge&logoColor=white&logo=uv&color=3776AB)
+
+To install this project, you will need to have Python and `pip` or `uv` installed on your machine.
 
-We recommend to use the node version specified in the `.nvmrc` file.
+We recommend using a virtual environment to manage your dependencies.
 
-**If you don't have `pnpm` installed, you can still use `npm` for all commands below, but we recommend to use `pnpm` for faster and more reliable installs.**
+**If you don't have `uv` installed, you can still use `pip` for all commands below, but we recommend using `uv` for faster and more reliable installs.**
```

Comment:
Not sure if the dev dependencies will be as easy to install with `pip` given they aren't under the conventional `[project.optional-dependencies]` that `pip` recognises, but rather under `[tool.uv]`. Personally would just suggest setting up `uv` if possible to avoif confusion here :)

Reply:
Sounds good